docs/api-docs/catalog/price-list-overview.md

```diff
@@ -42,7 +42,7 @@ The following OAuth Scopes are required:
 
 - `PriceRecordBatch`: A way to update several `Price Records` in a `Price List` at once. Using this bulk upsert endpoint, you can upsert up to 1000 Price Records in a single API call.
 
-- `Currency`:  A `Price List` can contain records for multiple currencies. At this time, *only price records that match the store's default currency will be used to determine storefront and in-cart prices.* Although BigCommerce supports a storefront currency selection, this is not currently integrated with Price Lists and will merely convert prices from the store's default currency for display convenience.
+- `Currency`:  A `Price List` can contain records for multiple currencies. At this time, *only price records that match the store's default currency will be used to determine storefront and in-cart prices.* Although BigCommerce supports a storefront currency selection, Multi-Currency storefronts are not currently integrated with Price Lists and will merely convert prices from the store's default currency for display convenience.

```

Comment:
Multi-currency storefronts that transact in currencies that are not the store default are integrated with Price Lists in those transactional currencies.  

Multi-currency storefronts that transact in the store default will use store default currency Price Lists and currency will be converted in the display.

Hope this is a bit more clear that my earlier comment @traci707